configs/hours.yaml

```diff
@@ -222,5 +222,9 @@ holidays:
     reason: Finals Week #no front desk staff
   - date: [2021-12-19, 2022-01-17]
     reason: Winter Break
-  - date: [2022-01-18, 2022-01-31]
+  - date: [2022-01-18, 2022-01-23]
     reason: Remote Instruction Closure
+  - date: [2022-01-24, 2022-01-30]
```

Comment:
maybe change this to end on 1/29 instead of 1/30? (I think right now it would overwrite the normal sunday closure with 9-6pm)